src/main/java/numberBaseball/ResultPrinter.java

```diff
@@ -0,0 +1,32 @@
+package numberBaseball;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import game.Result;
+
+@RequiredArgsConstructor
+public class ResultPrinter {
+    private final static String INPUT_MESSAGE = "숫자를 입력해주세요 : ";
+    private final static String BALL = "볼";
+    private final static String STRIKE = "스트라이크";
+
+    private final static String NOTHING = "낫싱";
+    @Getter
+    private final static String END_MESSAGE = "개의 숫자를 모두 맞히셨습니다! 게임 종료\n" +
```

Comment:
```suggestion
    private static final String INPUT_MESSAGE = "숫자를 입력해주세요 : ";
    private static final String BALL = "볼";
    private static final String STRIKE = "스트라이크";

    private static final String NOTHING = "낫싱";
    @Getter
    private static final String END_MESSAGE = "개의 숫자를 모두 맞히셨습니다! 게임 종료\n" +
```

문자열 관리 같은 부분은 
1차적으로는 관리적인 측면에서 효율적인 상수로 선언, 
상수로 관리해야 하는 필드가 늘어나는 경우 별도의 클래스 또는 enum으로 관리
보다 확장된 서비스를 고려한다면 key: value 기반의 파일로 관리하여 이를 매핑해주는 클래스를 생성하여 관리할 것 같습니다.

Reply:
final static을 static final로 변경하였습니다.  문자열들은 일단 Game클래스에 정적변수로 남겨두었습니다.